src/components/atoms/CheckBox.tsx

```diff
@@ -0,0 +1,21 @@
+import { CheckIcon } from '@/assets/icons';
+import React from 'react';
+
+const CheckBox: React.FC<{ active: boolean; onClick: () => void }> = ({
+  active = false,
+  onClick,
+}) => {
+  const commonStyle =
+    'flex justify-center items-center w-[20px] h-[20px] rounded-full';
+  return active ? (
+    <button onClick={onClick} className={`${commonStyle}  bg-primary2`}>
+      <CheckIcon />
+    </button>
+  ) : (
+    <button onClick={onClick} className={`${commonStyle} border border-gray5`}>
+      <CheckIcon stroke="#9299AA" />
+    </button>
+  );
+};
+
```

Comment:
이런 형태도 가능할 것 같네요..!🙇‍♂️

```suggestion
const CheckBox: React.FC<{ active: boolean; onClick: () => void }> = ({
  active = false,
  onClick,
}) => {
  const commonStyle = 'flex justify-center items-center w-20 h-20 rounded-full';
  const buttonClassName = `${commonStyle} ${active ? 'bg-primary2' : 'border border-gray5'}`;

  return (
    <button onClick={onClick} className={buttonClassName}>
      <CheckIcon stroke={active ? '기본값 혹은 undefined' : '#9299AA'} />
    </button>
  );
};

export default CheckBox;
```

Reply:
분기 파편화 이슈가 생길까봐 고민 했었는데 공유 주신 코드가 훨씬 간결해보이네요 👍🏻 
반영해두었습니다☺️ 감사해요!🙇‍♀️